### PR TITLE
[BOOT][DIRECTX][SHELL32] Unneeded semicolons (trivial)

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- Remove Unneeded semicolons

JIRA issue: [CORE-13162](https://jira.reactos.org/browse/CORE-13162)

## Proposed changes

- Remove Unneeded semicolons (except on 3rd party code)

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
Please wait before merge, i've  just found new ones

**Review by katahiromz — APPROVED** [review]

File: drivers/filesystems/ext2/src/read.c [hunk 1]
```diff
@@ -285,7 +285,7 @@ Ext2ReadVolume (IN PEXT2_IRP_CONTEXT IrpContext)
                         }
                     }
 
-                    Ext2CompleteIrpContext(IrpContext, Status);;
+                    Ext2CompleteIrpContext(IrpContext, Status);
```
Comment (HBelusca) [inline drivers/filesystems/ext2/src/read.c:288]: 3rd party code?

File: drivers/filesystems/reiserfs/src/read.c [hunk 2]
```diff
@@ -196,7 +196,7 @@ RfsdReadVolume (IN PRFSD_IRP_CONTEXT IrpContext)
                 }
 
                 BlockArray.Irp = NULL;
-                BlockArray.Lba = ByteOffset.QuadPart;;
+                BlockArray.Lba = ByteOffset.QuadPart;
```
Comment (HBelusca) [inline drivers/filesystems/reiserfs/src/read.c:199]: 3rd party code?

File: drivers/filesystems/ext2/src/read.c (hunk 1 above)
Comment (KyleKatarn31) [reply drivers/filesystems/ext2/src/read.c:288]: Do you want me to revert this one ? Or do we take it as it won't harm ?
I only took out code in 3rdparty folder

File: drivers/filesystems/reiserfs/src/read.c (hunk 2 above)
Comment (KyleKatarn31) [reply drivers/filesystems/reiserfs/src/read.c:199]: Do you want me to revert this one ? Or do we take it as it won't harm ?
I only took out code in 3rdparty folder

**Comment by KyleKatarn31** [comment]
@HBelusca : do you want me to revert on FS modules ?

File: drivers/filesystems/ext2/src/read.c (hunk 1 above)
Comment (SergeGautherie) [reply drivers/filesystems/ext2/src/read.c:288]: > 3rd party code?

Yes: see https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/README.FSD;hb=HEAD

> revert

Options are:
1. Cherry-pick from upstream, if already fixed there.
2. Use `__REACTOS__`, though a diff for such a minor change may be refused...
3. Disable warning (CMakeLists.txt) for the time being, if that applies.
4. Leave it as is :-/

File: drivers/filesystems/reiserfs/src/read.c (hunk 2 above)
Comment (KyleKatarn31) [reply drivers/filesystems/reiserfs/src/read.c:199]: Revert.

File: drivers/filesystems/ext2/src/read.c (hunk 1 above)
Comment (KyleKatarn31) [reply drivers/filesystems/ext2/src/read.c:288]: Revert...

**Comment by KyleKatarn31** [comment]
Reverted 3rd party code. @binarymaster @HBelusca @SergeGautherie : OK to go ?

**Review by SergeGautherie — APPROVED** [review]

**Pull request merged**